tools/codegen/base/trunk.json

```
@@ -0,0 +1,15 @@
+{
+  "repository": "https://github.com/trunk-rs/trunk",
+  "tag_prefix": "v",
+  "rust_crate": "${package}",
+  "asset_name": "${package}-${rust_target}.tar.gz",
+  "platform": {
```

```suggestion
  "version_range": ">= 0.19.0",
  "platform": {
```

Pre-0.19.0 versions don't have linux-musl binaries, so let's drop official support for them.